torch/_inductor/lowering.py

```diff
@@ -297,6 +297,26 @@ def wrapped(*args, **kwargs):
         return out
 
     aten_fn = get_overloads(aten_fn)
+    core_aten_ops = core_aten_decompositions()
+
+    for fn in aten_fn:
+        if not isinstance(fn, torch._ops.OpOverload):
+            continue
+
+        py_kernels = None
+        if hasattr(fn, "default"):
+            fn = fn.default
```

Comment:
What about the other overloads? I think we should do this for every overload of fn.